script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -83,45 +83,45 @@ a hand of unparalleled heat is unleashed towards Arcueid, melting the rubble in
 <Page24>
 "Arcueid―――!"
 In one instant.
-The tides of battle get turned in one instant.
+The tides of battle are reversed in an instant.
 <Page25>
-Arcueid said that she could withstand temperatures of up to 3000< |o>C.
-However, the 'hand' that melted the rubble with ease had a temperature of well over 3000< |o>C.
+Arcueid said that she could withstand temperatures of up to 3000℃.
+However, the 'hand' that melted the rubble with ease had a temperature well over that.
 <Page26>
-That combined with the other reasons for that dreadful feeling means that not even ashes will remain. Without an escape path, the flames became a tongue that torment its prey.
-The white figure burned away without a trace.
-―――That's what would have occurred if this happened last night.
+He had easily managed to accumulate three times that power. The flames become a tormenting tongue, playing with their prey who are unable to escape.
```

Comment:
Building on Gekimon's comment on the outdated file:
In the face of all of this, not even ashes can remain. The flames become a tormenting tongue, playing with their trapped prey.
The white figure burns away without a trace.
―――Or that's what would have occurred, if this had been last night.

Reply:
Okay so that one was a bit of a doozy, but actually not difficult. The correct subject eluded us.
The correct translation is:
"He could even stack/produce three of them, meaning that not even ashes will remain. The flames become a tormenting tongue, playing with their pray who are unable to escape" (or any variation thereof)